src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8333,6 +8339,38 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/action-constraints':
```

Comment:
I wondered about changing this to `GET /action-constraints/{constraintName}`?

Reply:
Actually, I that may be more suitable. I am trying to figure out a reason when we would want to use GET all constraints and it seems that we would rather need GET single by name.